### PR TITLE
Add Florentiner Recipe

### PR DESCRIPTION
**Pull request by martinaengel**
Maybe consider adding a Florentiner recipe. It is very delicious and great for Christmas season.

### CODE REVIEW

File: recipes/Florentiner.qmd [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - "Christmas"
+  - "Cookie"
+  - "Sweet"
+  - "Vegetarian"
+
+---
+
+## {{< fa basket-shopping >}} Ingredients
+
+For the shortcrust dough:
```
Comment (lnnrtwttkhn) [inline recipes/Florentiner.qmd:15]: Please add subheading, like this

```r
### For the shortcrust dough
```

File: recipes/Florentiner.qmd [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- [175 g] soft butter
```
Comment (lnnrtwttkhn) [inline recipes/Florentiner.qmd:16]: For all ingredients, please change to, e.g.,:

```r
- [ ] 175 g soft butter
```

The square brackets should be empty and only contain a space. In Markdown syntax, this then shows up as To-Do-List, like so:

- [ ] 175 g soft butter

**Review by lnnrtwttkhn — CHANGES_REQUESTED** [review]
Thanks for this PR @martinaengel! I've reviewed your PR and requested additional changes. Thank you!

**Comment by martinaengel** [comment]
Thank you for the comments! I implemented them in the changes!

File: recipes/Florentiner.qmd [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: "The best Florentiners"
+image: "https://heissehimbeeren.com/wp-content/uploads/2022/11/omas-florentiner-rezept-12.webp"
+image-alt: "https://images.lecker.de/klassische-florentiner/1x1,id=72f9c685,b=lecker,w=1600,h=,ca=12.50,0,87.50,100,rm=sk.jpeg"
+categories:
+  - Christmas
+  - Cookie
+  - Sweet
+  - Vegetarian
```
Comment (lnnrtwttkhn) [inline recipes/Florentiner.qmd:9]: Thank you @martinaengel for these additional changes! Would you you like to add your author name?
For this add your name to the author field in the YAMl header, like this:

```YAML
author: firstname lastname
```

**Review by lnnrtwttkhn — CHANGES_REQUESTED** [review]
Thank you for the additional changes @martinaengel! Do you want to add your author information.

**Comment by lnnrtwttkhn** [comment]
Thanks again @martinaengel, I will merge this PR now!

**Comment by lnnrtwttkhn** [comment]
@allcontributors please add @martinaengel for bug, content, code

**Pull request merged**